Testing/GameCardFuncTest.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+require_once 'gamecardfunc.php';
```

Comment:
Here you should require `'../gamecardfunc.php'` so you're getting the original file rather than the duplicate in your testing folder

Reply:
done